school/teacher.rb

```diff
@@ -1,10 +1,5 @@
 require_relative 'person'
 class Teacher < Person
-  def initialize(specialization)
-    @specialization = specialization
-    super()
-  end
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly previous reviewer mentioned about your `def`, please don't remove this part so constructor extends parent's constructar by adding `@specialization` and a `parameter` for it, like that🙏🏼 

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```